### ISSUE TITLE
Question about detailed training information

### ISSUE THREAD

**Issue by HYOJINPARK** [suhwan-cho/CRVOS#4]
Thanks for your great work
I try to reproduce your model, but unfortunately, I got below the accuracy.

Best 2017 J&F score 0.5339137622141097 
Best 2016 J&F score 0.7348164648522798 

The DAVIS 2017 is similar but DAVIS 2016 is different.

Could you tell me a detail train hyper-parameter?
   - Number of the batch in each GPU
   - Total number of GPU
   - Weight decay

**Comment by suhwan-cho** [comment]
Thanks for your attention to our work!

I replied to your email.
Hope you can reproduce the pre-trained model.
